### PR TITLE
Milestone 6

### PR DESCRIPTION
**Pull request by AbigiyaTY**
# Project deployed

In this milestone I:

- Deployed the project using the GitHub page.
- Added a live demo of the project that is deployed in the README file.

### CODE REVIEW

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @AbigiyaTY 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by PraisesPJMT** [comment]
## Amazing Project

I tested your deployed website and it looks great!
- I particularly love the clarity of content in your README.md file. It looks professional.
- No linter errors were observed which is excellent!
- You used the appropriate GitHub Flow


Happy coding!!

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 ## Built With
     HTML
     CSS
+## Live Demo
+https://abigiyaty.github.io/Portofolio-/
 
 👤 **Author**
 - GitHub: [@githubhandle](https://github.com/AbigiyaTY)
```
Comment (GoddessNgozi) [inline README.md:11]: - Good commit message!
- The link opens correctly to the website and the site layout is properly structured according to the template you choose. Good job! 🤗

**Review by GoddessNgozi — COMMENTED** [review]
Good Job🚀

**Comment by spydaspider** [comment]
The design looks good. Not many changes are required.

**Pull request merged**